### DISCUSSION TITLE
Does FreeTube have an algorithm that would recommend similar channels/videos?

### DISCUSSION THREAD

**Discussion by Adotac** [FreeTubeApp/FreeTube#4144] [General]
I'm still new in using FreeTube, and I had just imported my watch history and subscriptions. Now all I have in the subscription feed are just latest videos of my subscribed channels where some of them are not that interesting. It would be nice if FreeTube can analyze my watch history and curate an interesting feed for me.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
FT does not have system in place that recommends u videos. There is an open issue for it https://github.com/FreeTubeApp/FreeTube/issues/677 but its definitely a low priority right now
